### PR TITLE
New UI for NewNode VPN for Android

### PR DESCRIPTION
**Pull request by IvanSeleznevSpb**

### CODE REVIEW

File: android/vpn/app/src/main/java/com/clostra/newnode/vpn/statistics/StatisticsFragment.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        return new DataVolume(volume, volume / 2);
+    }
+
+    public static double floor(double value) {
+        BigDecimal bd = BigDecimal.valueOf(value);
+        bd = bd.setScale(2, RoundingMode.FLOOR);
+        return bd.doubleValue();
+    }
+
+    private String dateVolumeToString (double bytes) {
```
Comment (ghazel) [inline android/vpn/app/src/main/java/com/clostra/newnode/vpn/statistics/StatisticsFragment.java]: `android.text.format.Formatter.formatShortFileSize` does a great job at this.

File: android/vpn/app/src/main/java/com/clostra/newnode/vpn/VpnService.java [hunk 2]
```diff
@@ -12,10 +12,11 @@
 import android.os.ParcelFileDescriptor;
 import android.util.Log;
 import android.widget.Toast;
-import java.io.IOException;
 
 import com.clostra.newnode.NewNode;
 
+import java.io.IOException;
+
```
Comment (ghazel) [inline android/vpn/app/src/main/java/com/clostra/newnode/vpn/VpnService.java]: Was the change in this file necessary?

**Review by ghazel — CHANGES_REQUESTED** [review]

File: android/vpn/app/src/main/java/com/clostra/newnode/vpn/statistics/StatisticsFragment.java (hunk 1 above)
Comment (IvanSeleznevSpb) [reply android/vpn/app/src/main/java/com/clostra/newnode/vpn/statistics/StatisticsFragment.java]: Excellent. Thanks for the advice.

File: android/vpn/app/src/main/java/com/clostra/newnode/vpn/VpnService.java (hunk 2 above)
Comment (IvanSeleznevSpb) [reply android/vpn/app/src/main/java/com/clostra/newnode/vpn/VpnService.java]: I'm sorry. This is accidental. Revert.

**Review by ghazel — APPROVED** [review]

**Pull request merged**